api/v1alpha3/doc.go

```diff
@@ -18,4 +18,5 @@ limitations under the License.
 // +k8s:conversion-gen=sigs.k8s.io/cluster-api/api/v1beta1
 //
 // Deprecated: This package will be removed in one of the next releases.
+// +kubebuilder:unservedversion
```

Comment:
Looks like these package level markers are not doing anything. Can we drop them if not needed?

(Same for other packages)